src/features/landing/components/AboutUs/DeviceImage/DeviceImage.tsx

```diff
@@ -1,7 +1,32 @@
+import { Box } from '@mui/material';
+import Image from 'next/image';
 import React from 'react';
 
+import DeskTop from '../../../assets/img/AboutVilna/desktop.png';
+
 function DeviceImage() {
-  return <div>DeviceImage</div>;
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        justifyContent: 'center',
+        backgroundImage: `url('/assets/backGroundDesktop.png')`,
```

Comment:
this is not a good approach, what if the place of the image will be changed? then you will need to update the path in this line